dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsHelper.java

```diff
@@ -737,40 +680,99 @@ InputStream exportSecrets(final ExportSecretForm form, final User user)
      * @throws IOException
      * @throws DotDataException
      * @throws JSONException
-     * @throws DotSecurityException
-     * @throws EncryptorException
-     * @throws ClassNotFoundException
      */
     void importSecrets(final FormDataMultiPart multipart, final User user)
-            throws IOException, DotDataException, JSONException, DotSecurityException, EncryptorException, ClassNotFoundException {
+            throws IOException, DotDataException, JSONException {
+
+        processMultipart(multipart, new FileConsumer<Void>() {
+            private Key key = null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'key'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)